batch-import/post_content.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# POST a single video to a Box for segmentation and analysis
+# This requires that
+# 1. You have a Camio Box running and you can access it over the network
+# 2. You've registered a batch-import source with Camio
+# 3. That batch-import source is attached to the Box described in #1
```

Comment:
I mean that the batch-input camera must be connected (toggled on) to this Box in order for the webserver to be running and expecting videos to process.